contracts/cctp/events/SynapseCCTPEvents.sol

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+abstract contract SynapseCCTPEvents {
+    // TODO: figure out what we need to emit for the Explorer
+
+    /// @notice Emitted when a Circle token is sent with an attached action request.
+    /// @dev To fulfill the request, the validator needs to fetch `message` from `MessageSent` event
+    /// emitted by Circle's MessageTransmitter in the same tx, then fetch `signature` for the message from Circle API.
+    /// This data will need to be presented to SynapseCCTP on the destination domain,
+    /// along with `requestVersion` and `formattedRequest` emitted in this event.
+    /// @param destinationDomain    Domain of destination chain
+    /// @param nonce                Nonce of the CCTP message on origin domain
+    /// @param requestVersion       Version of the request format
+    /// @param formattedRequest     Formatted request for the action to take on the destination domain
+    /// @param kappa                Unique identifier of the request
+    event CircleRequestSent(
+        uint32 destinationDomain,
+        uint64 nonce,
+        uint32 requestVersion,
+        bytes formattedRequest,
+        bytes32 indexed kappa
```

Comment:
can change this variable name to be readeable unless it is dependent on the current bridge contracts and needs to be the same

Reply:
No need to be exactly the same, just reused the name by default. Will check the PRs to see where this change would lead to no merge conflicts, any preferences for the updated naming? Like `requestID` / `requestHash`?

Reply:
@aureliusbtc The name was adjusted in 25382d2.